README.md

```diff
@@ -1 +1,41 @@
-# To-Do-List
\ No newline at end of file
+# To-Do-List
+
+> The project focused on creating To do list structure
+
+## Built With
+- HTML
+- CSS
+-JAVASCRIPT
+​
+## Live Demo (if available)
+
+- https://alphantihinduka.github.io/Alpha-Mobile-portifolio/
+
```

Comment:
- [ ] (Optional) Kindly remove this link, it is the link for your portfolio project. Having a live demo link that redirects to the wrong page makes the `README.md` not professional. 👍